spec/integrations/base_spec.rb

```diff
@@ -58,6 +58,29 @@
       end
     end
 
+    context 'Given passed object is array-like' do
+      let(:blueprint) { blueprint_with_block }
+      let(:additional_object) { OpenStruct.new(obj_hash.merge(id: 2)) }
+      let(:obj) { Set.new([object_with_attributes, additional_object]) }
+
+      context 'and is an instance of a configured array-like class' do
+        before do
+          Blueprinter.configure { |config| config.array_like_classes = [Set] }
+        end
+        after { reset_blueprinter_config! }
+
+        it 'should return the expected array of hashes' do
+          should eq('[{"id":1,"position_and_company":"Manager at Procore"},{"id":2,"position_and_company":"Manager at Procore"}]')
+        end
+      end
+
+      context 'and is not an instance of a configured array-like class' do
+        it 'should raise an error' do
+          expect { blueprint.render(obj) }.to raise_error(NoMethodError)
```

Comment:
there's not really anything else to assert here.  We already have tests for that cover a hash being handled properly, and hashes are "array-like" in that they are `Enumerable`, so here I just mimicked what would happen if you pass it a collection for which you didn't pass the class of to `array_like_classes`.